utils/build-script-impl

```diff
@@ -1668,7 +1662,7 @@ for host in "${ALL_HOSTS[@]}"; do
             exit 1
         fi
     else
-        CLANG_BIN="$(build_directory_bin ${LOCAL_HOST} llvm)"
+        CLANG_BIN="$(build_directory_bin ${LOCAL_HOST} libcxx)"
```

Comment:
What's the reason for changing this to `libcxx`?

Reply:
No reason now that I realized the `llvm` component itself must be kept. I just forgot to change them back — thanks.